### PR TITLE
SerialManager: Quote Popen annotation

### LINKED ISSUE #1118: Popen annotation causes runtime error
```python
Traceback (most recent call last):
  File "/usr/bin/blueman-applet", line 42, in <module>
    BluemanApplet()
  File "/usr/lib/python3.7/site-packages/blueman/main/Applet.py", line 34, in __init__
    self.Plugins.load_plugin()
  File "/usr/lib/python3.7/site-packages/blueman/main/PluginManager.py", line 92, in load_plugin
    importlib.import_module(self.module_path.__name__ + ".%s" % plugin)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/lib/python3.7/site-packages/blueman/plugins/applet/SerialManager.py", line 18, in <module>
    class SerialManager(AppletPlugin):
  File "/usr/lib/python3.7/site-packages/blueman/plugins/applet/SerialManager.py", line 38, in SerialManager
    scripts: Dict[str, Dict[str, Popen[str]]] = {}
TypeError: 'type' object is not subscriptable
```

Quick fix to make it run below but I am not sure this is the right way.

```patch
diff --git a/blueman/plugins/applet/SerialManager.py b/blueman/plugins/applet/SerialManager.py
index 79769e12..c089c190 100644
--- a/blueman/plugins/applet/SerialManager.py
+++ b/blueman/plugins/applet/SerialManager.py
@@ -7,7 +7,6 @@ from blueman.gui.Notification import Notification
 from blueman.Sdp import SERIAL_PORT_SVCLASS_ID
 from blueman.services.Functions import get_services
 from _blueman import rfcomm_list
-from subprocess import Popen
 import logging
 import os
 import signal
@@ -35,7 +34,7 @@ class SerialManager(AppletPlugin):
                              "Upon device disconnection the script will be sent a HUP signal</span>")},
     }
 
-    scripts: Dict[str, Dict[str, Popen[Any]]] = {}
+    scripts: Dict[str, Dict[str, "Popen[Any]"]] = {}
 
     def on_load(self):
         self.scripts = {}
```

### PR DESCRIPTION
**Pull request by infirit**
There is a disconnect between typeshed and the std lib annotation.
fixes #1118

TypeError: 'type' object is not subscriptable

### CODE REVIEW

File: blueman/plugins/applet/SerialManager.py [hunk 1]
```diff
@@ -7,7 +7,6 @@
 from blueman.Sdp import SERIAL_PORT_SVCLASS_ID
 from blueman.services.Functions import get_services
 from _blueman import rfcomm_list
-from subprocess import Popen
```
Comment (cschramm) [inline blueman/plugins/applet/SerialManager.py]: We need that import in `call_script`.

File: blueman/plugins/applet/SerialManager.py [hunk 2]
```diff
@@ -35,7 +34,7 @@ class SerialManager(AppletPlugin):
                              "Upon device disconnection the script will be sent a HUP signal</span>")},
     }
 
-    scripts: Dict[str, Dict[str, Popen[Any]]] = {}
+    scripts: Dict[str, Dict[str, "Popen[Any]"]] = {}
```
Comment (cschramm) [inline blueman/plugins/applet/SerialManager.py]: The `Any` import is technically unused now.

Comment (infirit) [reply blueman/plugins/applet/SerialManager.py]: mypy still wants it.

Comment (cschramm) [reply blueman/plugins/applet/SerialManager.py]: Ok, depends on the context... 🙈

**Comment by infirit** [comment]
fml, mypy wants the Any import but then flake objects as it is unused.. hopefully everyone is happy now..

**Pull request merged**